### PR TITLE
[MIG] base_menu_visibility_restriction: Migration to 17.0

### PR DESCRIPTION
**Pull request by nedaszilinskas**

### CODE REVIEW

**Review by ioans73 — APPROVED** [review]
Functional review

File: base_menu_visibility_restriction/demo/ir_ui_menu.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="base.module_mi" model="ir.ui.menu">
```
Comment (jeroenev) [inline base_menu_visibility_restriction/demo/ir_ui_menu.xml]: This is a non-existent module, installing the module with demo-data enabled will cause an error

Comment (nedaszilinskas) [reply base_menu_visibility_restriction/demo/ir_ui_menu.xml]: @jeroenev good point. I removed it.

File: base_menu_visibility_restriction/demo/res_group.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="group_demo_user" model="res.groups">
+        <field name="name">Menu visibility restriction demo group</field>
+        <field eval="[(4, ref('base.user_demo'))]" name="users" />
+    </record>
+</odoo>
```
Comment (Tisho99) [inline base_menu_visibility_restriction/demo/res_group.xml]: I think this file is not needed due to the elimination of the ir_ui_menu.xml file.

Now the new group does nothing

File: base_menu_visibility_restriction/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    "category": "Extra Tools",
+    "development_status": "Production/Stable",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "summary": "Restrict (with groups) menu visibilty",
+    "website": "https://github.com/OCA/server-ux",
+    "license": "AGPL-3",
+    "depends": ["web_tour"],
+    "maintainers": ["victoralmau"],
+    "data": ["views/ir_ui_menu.xml"],
+    "demo": ["demo/res_group.xml"],
```
Comment (Tisho99) [inline base_menu_visibility_restriction/__manifest__.py]: I think this file is not needed due to the elimination of the ir_ui_menu.xml file.

Now the new group does nothing

**Review by Tisho99 — CHANGES_REQUESTED** [review]
LGTM.

Minor suggestions

**Review by CasVissers-360ERP — APPROVED** [review]
Functional review

**Review by Tisho99 — APPROVED** [review]
LGTM. Minor suggestions

**Comment by HaraldPanten** [comment]
@nedaszilinskas could you rebase, please?

THX

File: base_menu_visibility_restriction/__manifest__.py (hunk 3 above)
Comment (nedaszilinskas) [reply base_menu_visibility_restriction/__manifest__.py]: @Tisho99 I removed it. Thanks!

**Comment by nedaszilinskas** [comment]
@HaraldPanten I rebased it.

File: base_menu_visibility_restriction/__manifest__.py (hunk 3 above)
Comment (Tisho99) [reply base_menu_visibility_restriction/__manifest__.py]: Thanks to you!

**Review by luis-ron — APPROVED** [review]
Functional Review: LGTM 👍🏻

**Review by HaraldPanten — APPROVED** [review]

**Comment by HaraldPanten** [comment]
@etobella could we merge this one, please?

THX!

**Review by etobella — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**